src/site/pages/digital/src/containers/SelectionPopup/modules/BusButtonModule.tsx

```diff
@@ -13,23 +16,83 @@ type Props = {
 export const BusButtonModule = ({ info }: Props) => {
     const { renderer, history } = info;
 
-    const [props, ps] = useSelectionProps(
+    const [props, items] = useSelectionProps(
         info,
-        (s): s is (InputPort | OutputPort) => (
-            (s instanceof InputPort && s.getWires().length === 0) ||
-            s instanceof OutputPort
+        (s): s is (Component | InputPort | OutputPort) => (
+            (s instanceof Component ||
+             s instanceof InputPort && s.getWires().length === 0) ||
+             s instanceof OutputPort && s.getWires().length === 0
```

Comment:
The `&&` operator executes before the `||`, so this here works fine as is. For clarity, I think there should be parenthesis around each of the `&&` operands. Also, the parenthesis there does not have to go around `s instanceof Component`.
Basically, this block should look like:
```ts
s instanceof Component ||
(s instanceof InputPort && s.getWires().length === 0) ||
(s instanceof OutputPort && s.getWires().length === 0)
```